js/reference.js

```diff
@@ -321,7 +359,37 @@ var ERMrest = (function(module) {
                 // this can probably be direct calls to the module._http
                 // I do not think we need to re-use the current ...entity.get(...)
                 // methods implemented in the other scripts
-                notimplemented();
+
+                // TODO: get the server
+                var uri = server.uri + "/catalog/" + this._catalogId + "/schema/" +
+                    module._fixedEncodeURIComponent(this._schemaName) + ":" +
+                    module._fixedEncodeURIComponent(this._table.name);
+                    // TODO: filters
+
+
+                return module._http.get(uri).then(function success(response) {
```

Comment:
okay but this is the part that you were going to fix to only get back the `limit` tuples, right?